distribution/docs/src/main/resources/_contents/_data/json-definition-intro-contents.adoc

```diff
@@ -0,0 +1,35 @@
+
+=== JSON "Definition" Files.
+
+[WARNING]
+====
+This section concerns capabilities that are considered experimental. The features described in this section may change or be removed in a future version of the application.
+====
+
+${branding} supports adding new attribute types, metacard types, validators, and more using json-formatted definition files.
+
+The following may be defined in a JSON definition file:
+
+- <<_attribute_type_definition_file,Attribute Types>>
+- <<_metacard_type_definition_file,Metacard Types>>
+- <<_global_attribute_validators_file,Global Attribute Validators>>
+- <<_default_attribute_values,Default Attribute Values>>
+- <<_attribute_injections,Attribute Injections>>
+
+==== Definition File Format
+
+A definition file follows the JSON format as specified in http://www.ecma-international.org/publications/standards/Ecma-404.htm[ECMA-404].
+All definition files must be valid JSON in order to be parsed.
+
+A single definition file may define as many of the types as needed.
+This means that types can be defined across multiple files for grouping or clarity.
+
+==== Deploying Definition Files
+
+The file must have a `.json` extension in order to be picked up by the deployer.
+Once the definition file is ready to be deployed, put the definition file `<filename>.json` into the `etc/definitions` folder.
+
+Definition files can be updated while they are in the `etc/definitions` folder.
```

Comment:
Perhaps mention that the file can be dynamically added, updated, and deleted, and that a restart is not required, but will be immediately applied. Also, multiple .json definitions files can be added.